src/aere_parse.erl

```diff
@@ -125,8 +125,14 @@ commands() ->
     ].
 
 
--spec resolve_command(atom()) -> command_spec() | undefined.
+-spec resolve_command(string() | binary() | atom()) -> command_spec() | undefined.
```

Comment:
Why did this start to accept binaries and strings?

Reply:
Because user inputs are binary and it does not hurt to accept strings either